### DISCUSSION TITLE
What can we call the gh-action?

### DISCUSSION THREAD

**Discussion by epijim** [insightsengineering/r-pkg-installation-validation#5] [Q&A] [answered]
Some options - assuming `r-pkg-` is ok:

- r-pkg-installation-validation
- r-pkg-validation
- r-pkg-installation

Currently it's `r-pkg-installation-validation` as it's the validation at installation, as it's just very clearly documenting the state of the package at installation. I have switched it away from `r-pkg-validation`, as we'd usually align validation more with the use of the package - which includes topics that can't be purely extracted from the code (e.g. we want to make sure documentation, or unit testing, is sufficient - not just present). And it would be validated for a specific SCE - but in our case we can't predict every SCE make up, so it's validated in a quite generic environment.

Is there a name that better captures what this gh-action actually is?

**Comment by dgkf** [comment, marked as answer]
I think it's fair to just go for `r-pkg-validation`, recognizing that there are gaps, but that this is intended to be the closest we can get in a publicly distributed generic form.

**Reply by epijim** [reply]
Renamed it `thevalidatoR` to make the hex sticker easier to design for gDesign :)
